.github/renovate-config.js

```diff
@@ -0,0 +1,43 @@
+const helmRegex = {
+  customType: "regex",
+  datasourceTemplate: "helm",
+  matchStringsStrategy: "combination",
+};
+
+const customRegex = {
+  customType: "regex",
+  matchStringsStrategy: "combination",
+};
+
+module.exports = {
+  username: "renovate[bot]",
+  gitAuthor: "Renovate Bot <bot@renovateapp.com>",
+  onboarding: false,
+  platform: "github",
+  dryRun: null,
+  repositories: ["fluent/fluent-operator"],
+  enabledManagers: ["custom.regex"],
+  extends: ["config:recommended"],
+  customManagers: [
+    {
+      customType: "regex",
+      matchStringsStrategy: "any",
+      fileMatch: [
+        "charts/fluent-operator/values.yaml",
+        "config/samples/fluentbit_v1alpha2_fluentbit.yaml",
+        "docs/best-practice/forwarding-logs-via-http/deploy/fluentbit-fluentBit.yaml",
+        "manifests/kubeedge/fluentbit-fluentbit-edge.yaml",
+        "manifests/logging-stack/fluentbit-fluentBit.yaml",
+        "manifests/quick-start/fluentbit.yaml",
+        "manifests/regex-parser/fluentbit-fluentBit.yaml",
+        "cmd/fluent-watcher/fluentbit/VERSION",
```

Comment:
```suggestion
        "charts/fluent-operator/values.yaml",
        "config/**/*.yaml",
        "docs/**/*.yaml",
        "manifests/**/*.yaml",
        "cmd/fluent-watcher/fluentbit/VERSION",
```

To make things less sensitive for file renames and such.